web/src/assets/base.css

```
@@ -58,12 +232,194 @@ html.light {
+@keyframes pulse {
+  0%,
+  100% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.3;
+  }
+}
+
+/* Kill-feed toast animation */
+@keyframes slideUp {
```

Stylelint (17.9.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)